src/glass/src/app/shared/services/status-route-guard.service.ts

```diff
@@ -11,31 +25,39 @@ import * as _ from 'lodash';
 import { Observable, of } from 'rxjs';
 import { catchError, map } from 'rxjs/operators';
 
-import { Status, StatusService, StatusStageEnum } from '~/app/shared/services/api/status.service';
+import {
+  LocalNodeService,
+  NodeStatus,
+  StatusStageEnum
```

Comment:
It seems like it needs to be applied to https://github.com/aquarist-labs/aquarium/blob/main/src/glass/src/app/shared/services/status-route-guard.service.spec.ts as well:

```
diff --git a/src/glass/src/app/shared/services/status-route-guard.service.spec.ts b/src/glass/src/app/shared/services/status-route-guard.service.spec.ts
index 5c26f0f..528a7bf 100644
--- a/src/glass/src/app/shared/services/status-route-guard.service.spec.ts
+++ b/src/glass/src/app/shared/services/status-route-guard.service.spec.ts
@@ -5,7 +5,8 @@ import { ActivatedRouteSnapshot, Router, RouterStateSnapshot, UrlTree } from '@a
 import { RouterTestingModule } from '@angular/router/testing';
 import { of } from 'rxjs';
 
-import { StatusService, StatusStageEnum } from '~/app/shared/services/api/status.service';
+import { StatusStageEnum } from '~/app/shared/services/api/local.service';
+import { StatusService } from '~/app/shared/services/api/status.service';
 import { StatusRouteGuardService } from '~/app/shared/services/status-route-guard.service';
 
 const fakeRouterStateSnapshot = (url: string): RouterStateSnapshot =>
```

And I think there is another change needed, because the tests are still failing on my system. I'm not (yet) sure what the issue is.